services.yaml

```diff
@@ -3172,7 +3140,29 @@ definitions:
       initiator_type:
         type: string
         example: "User"
-  ServiceBasic:
+  EnvironmentWithRole:
+    type: object
+    description: Extended information about a environment key
+    properties:
+      EnvironmentBasic:
```

Comment:
i'm not sure this does what you mean, you need a $ref
look at Event / EventStats

Reply:
didn't work in spectacle

Reply:
Let's start with this one and when additional option will be added to ServiceBasic, will change to #ref